### PR TITLE
Added Endpoint Config to Custom Actions Doc

### PR DESCRIPTION
**Pull request by b-quachtran**
**Proposed changes**:
- Added snippet on endpoint.yml config in custom actions doc

**Status (please check what you already did)**:
- [ ] added some tests for the functionality
- [x] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/master/changelog) for instructions)
- [ ] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Comment by akelad** [comment]
@erohmensing @melindaloubser1 you two have a better overview of the actions restructure - does that make sense to put it in here?

**Comment by melindaloubser1** [comment]
It could go here, but based on where all other references to `endpoints.yml` are, this could go be under `Architecture` - Even if brief, should there be a `Action Servers` page under there that specifies this & links to the Action Server docs?

File: docs/docs/custom-actions.mdx [hunk 1]
```diff
@@ -5,7 +5,7 @@ title: Custom Actions
 abstract: A custom action can run any code you want, including API calls, database queries etc. They can turn on the lights, add an event to a calendar, check a user's bank balance, or anything else you can imagine.
 ---
 
-For details on how to implement a custom action, see the [SDK documentation](https://rasa.com/docs/action-server/running-action-server).
+For details on how to implement a custom action and how to run a custom action server see the [action server documentation](https://rasa.com/docs/action-server).
 Any custom action that you want to use in your stories should be added into the
```
Comment (erohmensing) [inline docs/docs/custom-actions.mdx:9]:
```suggestion

Any custom action that you want to use in your stories should be added into the
```

File: docs/docs/custom-actions.mdx [hunk 2]
```diff
@@ -52,3 +52,15 @@ Your action server should respond with a list of events and responses:
   "responses": [{}]
 }
 ```
+
+## Configuring the Actions Server URL
```
Comment (erohmensing) [inline docs/docs/custom-actions.mdx:56]:
```suggestion
## Configuring the Action Server URL
```

**Review by erohmensing — APPROVED** [review]

File: docs/docs/custom-actions.mdx [hunk 3]
```diff
@@ -52,3 +52,15 @@ Your action server should respond with a list of events and responses:
...
+
+To connect Rasa Open Source to your actions server, add the URL to your `endpoints.yml`:
+
+```yaml-rasa title="endpoints.yml"
+action_endpoint:
+  url: http://localhost:5055/webhook
+```
+
+
```
Comment (erohmensing) [inline docs/docs/custom-actions.mdx:65]:
```suggestion
```

**Comment by erohmensing** [comment]
Sorry, I reviewed this code before reading the comments lol. @melindaloubser1 I think your comment makes sense, as that's where all the endpoints currently live

**Pull request closed without merging**